src/puppetlabs/ezbake/core.clj

```diff
@@ -427,7 +442,8 @@ Dependency tree:
                                                           "puppetlabs.trapperkeeper.main")
          :java-args                 (get-local-ezbake-var lein-project :java-args
                                                           "-Xmx192m")
-         :bootstrap-source          (get-local-ezbake-var lein-project :bootstrap-source :file)}))))
+                                    ; Convert to string so ruby doesn't barf on the hyphens
+         :bootstrap-source          (name (get-local-ezbake-var lein-project :bootstrap-source :bootstrap-cfg))}))))
```

Comment:
any reason not to call your validation function here instead of during the action?  seems like a good practice to keep config initialization and validation as close together as possible?